nixos/modules/services/matrix/matrix-sliding-sync.nix

```
@@ -1,10 +1,18 @@
 { config, lib, pkgs, ... }:
 
 let
-  cfg = config.services.matrix-synapse.sliding-sync;
+  cfg = config.services.matrix-sliding-sync;
 in
 {
-  options.services.matrix-synapse.sliding-sync = {
+  imports = [
+      (lib.mkRenamedOptionModuleWith {
+        sinceRelease = 2405;
+        from = [ "services" "matrix-synapse" "sliding-sync" ];
+        to = [ "services" "matrix-sliding-sync" ];
+      })
```

```suggestion
      (lib.mkRenamedOption [ "services" "matrix-synapse" "sliding-sync" ] [ "services" "matrix-sliding-sync" ])
```
we can just do this and then could then also drop the release notes entry since there is really not action needed.

---

I've updated my branch.